src/components/HomePage/UpcomingEvents.js

```diff
@@ -29,29 +44,44 @@ function UpcomingEvents() {
 	const currentTime = new Date();
 	let eventsDisplayed = 0;
 	const numEventsToDisplay = 2;
-	const timeSlots = [];
+
+	const events = [];
 	// Assumes workshopSchedule is ordered by time
 	for (let i = 0; i < workshopSchedule.length && eventsDisplayed < numEventsToDisplay; i++) {
-		const workshop = workshopSchedule[i];
-		if (currentTime < workshop.startTime) {
-			timeSlots.push(workshop);
-			eventsDisplayed += workshop.events.length;
+		const timeSlot = workshopSchedule[i];
+		if (currentTime < timeSlot.startTime) {
+			for (const event of timeSlot.events) {
+				events.push({
+					startTime: `${timeFormatter.format(timeSlot.startTime)} ${currentTimeZoneShort}`,
```

Comment:
If we are using a `Intl.DateTimeFormat` already, then just add `timeZoneName: 'short'` to the DateTimeFormat options, and don't bother getting `currentTimeZoneShort` from the `timezone_names` package.

Reply:
Resolved now with @Timthetic's commit